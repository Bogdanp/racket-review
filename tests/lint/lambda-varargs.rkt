#lang racket/base

(lambda xs
  (void))

(lambda (a b . xs)
  (void))

(lambda _
  (void))
