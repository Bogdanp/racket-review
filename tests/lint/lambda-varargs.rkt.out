/lambda-varargs.rkt:3:9:warning:identifier 'xs' is never used
/lambda-varargs.rkt:6:10:warning:identifier 'a' is never used
/lambda-varargs.rkt:6:12:warning:identifier 'b' is never used
/lambda-varargs.rkt:6:16:warning:identifier 'xs' is never used
/lambda-varargs.rkt:9:9:warning:avoid lambda _
